packages/host/src/lib/host.ts

```diff
@@ -122,6 +122,7 @@ export class Host implements IComponent {
         const prettyLog = new DataStream().map(prettyPrint({ colors: this.config.logColors }));
 
         this.logger.addOutput(prettyLog);
+        this.serviceDiscovery.logger.addOutput(prettyLog);
```

Comment:
this.serviceDiscovery.logger.pipe(this.logger)

Reply:
good catch